src/tests/Microsoft.PowerFx.Core.Tests/PowerFxTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Globalization;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public abstract class PowerFxTest
+    {
+        public PowerFxTest()
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

Can we add a comment here like:

We would like to use ModuleInitializer, but that requires .Net 5 and the tests can't run on that version yet. 
[https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.moduleinitializerattribute?view=net-6.0](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.moduleinitializerattribute?view=net-6.0)